webview/src/experiments/components/Experiments.tsx

```diff
@@ -67,34 +69,55 @@ const getDefaultColumn = () =>
     size: 240
   })
 
+const collectColumnObj = (
+  columns: Column[]
+  // eslint-disable-next-line sonarjs/cognitive-complexity
+): { [parentPath: string]: Column[] } => {
+  const columnObj: { [parentPath: string]: Column[] } = {}
```

Comment:
[I] Move this back into the extension. Only send minimal columns data to the webview.

[C] Consider limiting the amount of data we send for each experiment to the required columns.

Reply:
Also need to work out how to virtualize columns. Rows seems easy enough